lib/phoenix/live_dashboard/pages/ecto_stats_page.ex

```diff
@@ -272,23 +263,35 @@ defmodule Phoenix.LiveDashboard.EctoStatsPage do
     {sorted, length(rows)}
   end
 
-  defp format(_, %struct{} = value) when struct in [Decimal, Postgrex.Interval],
-    do: struct.to_string(value)
+  defp render_value(%{value: %struct{}} = assigns) when struct in [Decimal, Postgrex.Interval] do
+    ~H"""
+    <%= @value.__struct__.to_string(@value) %>
```

Comment:
Instead of calling this function `render_value` as a component, I think it should be called `<%= format(type, value) %>` as before. I don't think there is a need to force this as a component, it only makes things a bit more verbose. :)